hardware/tests/test_web_client.py

```diff
@@ -4,9 +4,8 @@
 from requests.exceptions import HTTPError
 
```

Comment:
this is minor, but please update the class LanClientTests to be renamed WebClientTests so that it matches.

Reply:
My bad. Fixed

Reply:
Fixed.